src/tito/cli.py

```diff
@@ -654,9 +654,7 @@ def main(self, argv):
             error_out("Tagging has been disabled in this git branch.")
 
         tagger_class = None
-        if self.options.use_version:
-            tagger_class = get_class_by_name("tito.tagger.ForceVersionTagger")
-        elif self.config.has_option("buildconfig", "tagger"):
+        if self.config.has_option("buildconfig", "tagger"):
```

Comment:
So this definitely looks more logical, my only concern is changing behavior that might catch someone off guard. Right now anyone using one of the other three bundled taggers, or a custom one in their repo, *might* be using --use-version and expecting it to override their normal versioning mechanism but instead the normal tagger runs and does nothing with that info.

So any ideas how we could roll it out as smoothly as possible? 

We could add a "supports_manual_version" method on the tagger base class, default it to False there, then set to true just for the main VersionTagger, and then make absolutely sure an attempt to use another tagger with --use-version will fail, and we could add support to other taggers as needed, and anyone who tries will get a clear error about it.

A unit test to this effect would be awesome too.

I'm open to other options though but I would like to find someway to make sure we don't trigger the above scenario.